.gitattributes

```diff
@@ -0,0 +1 @@
+.sql linguist-generated=true
```

Comment:
I think maybe the pattern is wrong (https://www.git-scm.com/docs/gitignore#_pattern_format), the files are still showing. When in doubt on how to do stuff and you can't find clear documentation is good to look at examples of other people. Here are some that might give you a hint how this should be set: 
- https://github.com/silahian/VisualHFT/blob/028d50bc8a67daa96d31a783c7417fae01b94ea6/.gitattributes#L5
- https://github.com/helpwave/services/blob/dac9a5fe78bbe0fdb3497d4995fd37a2563a98dc/.gitattributes#L12
- The full search: https://github.com/search?q=sql+linguist-generated%3Dtrue+path%3A.gitattributes&type=code


A good practice is to review yourself the pr before publishing it to check that it shows as intended and also helps sometimes detect nits or mistakes that would save you a future comment from one reviewer.

Reply:
OKay thanks. I will update

Reply:
I have updated it in this commit https://github.com/izuchukwumba/peered/pull/18/commits/27c7443539a37c234d407910f1468db115188954